Week1/homework/exercise1/server.js

```diff
@@ -0,0 +1,133 @@
+let mysql = require("mysql");
+const employees = require("./employees");
+const departments = require("./departments");
+const projects = require("./projects");
+
+const queryTablesArray = [
+    "CREATE TABLE IF NOT EXISTS Employee(emp_no INT NOT NULL AUTO_INCREMENT, emp_name Varchar(25) NOT NULL, salary Double NOT NULL," +
+    "reports_to Varchar(25) NOT NULL, PRIMARY KEY(emp_no));",
+    "CREATE TABLE IF NOT EXISTS Departments(dept_no INT NOT NULL AUTO_INCREMENT, dept_name Varchar(25) NOT NULL," +
+    "manager Varchar(25) NOT NULL, PRIMARY KEY(dept_no));",
+    "CREATE TABLE IF NOT EXISTS Projects(proj_no INT NOT NULL AUTO_INCREMENT, proj_name Varchar(25) NOT NULL," +
+    "starting_date Date NOT NULL, ending_date Date NULL, PRIMARY KEY(proj_no));"
+];
+
+const messagesArray = [
+    "Table Employee successfully created",
+    "Table Departments successfully created",
+    "Table Projects successfully created"
+];
+
+function createDatabase(database_name){
+    connection = mysql.createConnection({
+        host : "localhost",
+        user : "root",
+        password : "password",
+        database : "world"
+    });
+
+    connection.connect(err => {
+        if (err) {
+          return console.error('error: ' + err.message);
+        }
+       
+        console.log('Connected Successfully')
+    });
+
+    connection.query(
+        `CREATE SCHEMA IF NOT EXISTS ${database_name}`, (error, results, fields) => {
+        if(error) throw error;
+        console.log("Database Company successfully created");
+        createTables("company");
```

Comment:
:+1: for doing each step (create database, create table, populate with data) in the callback of the previous step!